app/controllers/announcements_controller.rb

```diff
@@ -10,7 +10,7 @@ class AnnouncementsController < ApplicationController
   action_auth_level :index, :instructor
   def index
     @announcements = if @cud.user.administrator?
-                       Announcement.where("course_id=? or system", @course.id)
+                       Announcement.where("course_id=? or `system`", @course.id)
```

Comment:
definitely seems like a newly added keyword, at least in mysql. https://dev.mysql.com/doc/refman/8.0/en/keywords.html